cmd/serv.go

```diff
@@ -296,6 +296,13 @@ func runServ(c *cli.Context) error {
 		gitcmd = exec.Command(verb, repoPath)
 	}
 
+	if isWiki {
+		err = repo.InitWiki()
+		if err != nil {
```

Comment:
Also a good idea to contain them (not that it matters _here_, but it's good practice)